test/extended/syncer/syncer.go

```diff
@@ -0,0 +1,79 @@
+package syncer
+
+import (
+	"os"
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	apb "github.com/kcp-dev/kcp-tests/test/extended/apibinding"
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[area/transparent-multi-cluster]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithWorkSpace("kcp-syncer")
+	)
+
+	g.It("Author:pewang-Critical-[Smoke][BYO] Validate create, modifying, deleting a deployment from KCP gets synced to the pcluster", func() {
+		pclusterKubeconfig := os.Getenv("PCLUSTER_KUBECONFIG")
+		if pclusterKubeconfig == "" {
+			g.Skip("None pcluster kubeconfig set for the test scenario")
+		}
+		myWs := k.WorkSpace()
+		k.SetGuestKubeconf(pclusterKubeconfig)
+
+		g.By("# Create workload sync and generate syncer resources manifests in current workspace")
+		mySyncer := NewSyncTarget()
+		mySyncer.OutputFilePath = "/tmp/" + myWs.Name + "." + mySyncer.Name + ".yaml"
+		mySyncer.Create(k)
+		defer mySyncer.Clean(k)
+
+		g.By("# Apply syncer resources on pcluster and wait for synctarget become ready")
+		defer k.AsGuestKubeconf().WithoutNamespace().WithoutWorkSpaceServer().Run("delete").Args("-f", mySyncer.OutputFilePath).Execute()
+		err := k.AsGuestKubeconf().WithoutNamespace().WithoutWorkSpaceServer().Run("apply").Args("-f", mySyncer.OutputFilePath).Execute()
+		o.Expect(err).ShouldNot(o.HaveOccurred())
+		mySyncer.WaitUntilReady(k)
+		mySyncer.CheckDisplayAttributes(k)
```

Comment:
@Phaow verify that the below columns are present as well along with Name & AGE and they are not null, thanks !! There was a bug where SYNCED API RESOURCES was not indicating anything.

[knarra@knarra ~]$ kubectl get synctarget -o wide
NAME         LOCATION     READY   SYNCED API RESOURCES   KEY                                      AGE
qecluster1   qecluster1   False                          cfKajMFOBq2wSWpukT9xyoCBcSqs1NRRFAaytK   4d3h